### PR TITLE
🌐 German translations (thanks to thewolfofallstreetz on the Ghost forum)

### PR DESCRIPTION
**Pull request by cathysarisky**
ref: https://forum.ghost.org/t/help-translate-ghost-beta/37461/113

Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [x] There's a clear use-case for this code change, explained below
- [x] Commit message has a short title & references relevant issues
- [x] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution!

### CODE REVIEW

File: ghost/i18n/locales/de/portal.json [hunk 1]
```diff
@@ -25,12 +25,12 @@
     "Cancel subscription": "Abonnement kündigen",
     "Cancellation reason": "Kündigungsgrund",
     "Change": "Ändern",
-    "Change plan": "",
+    "Change plan": "Tarif änder",
```
Comment (lsinger) [inline ghost/i18n/locales/de/portal.json:28]: @ronaldlangeveld this looks like a typo — should be `Tarif ändern` I'd say (except there's some context that I don't have that adds an `n`).

File: ghost/i18n/locales/de/portal.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "(save {{highestYearlyDiscount}}%)": "",
+    "(save {{highestYearlyDiscount}}%)": "(jetzt sparen {{highestYearlyDiscount}}%)",
```
Comment (lsinger) [inline ghost/i18n/locales/de/portal.json:2]: @ronaldlangeveld should be `(jetzt {{highestYearlyDiscount}}% sparen)`

("save now 20%" vs. "save 20% now")

(Sorry for jumping into this merged PR, just found it in an open tab and thought I'd share my two native speaker cents.)

File: ghost/i18n/locales/de/portal.json [hunk 3]
```diff
@@ -85,7 +85,7 @@
     "Manage": "Verwalten",
     "Maybe later": "Vielleicht später",
     "Memberships unavailable, contact the owner for access.": "",
-    "month": "",
+    "month": "monat",
```
Comment (lsinger) [inline ghost/i18n/locales/de/portal.json:88]: Should be capitalized as it's a noun: `Monat`

File: ghost/i18n/locales/de/portal.json [hunk 4]
```diff
@@ -160,7 +160,7 @@
     "Welcome to {{siteTitle}}": "Willkommen bei {{siteTitle}}",
     "When an inbox fails to accept an email it is commonly called a bounce. In many cases, this can be temporary. However, in some cases, a bounced email can be returned as a permanent failure when an email address is invalid or non-existent.": "Wenn ein Posteingang eine E-Mail nicht annimmt, spricht man von einem Bounce. In vielen Fällen kann dies vorübergehend sein (Soft Bounce). In einigen Fällen kann eine E-Mail jedoch als dauerhafter Fehler zurückgeschickt werden, wenn eine E-Mail-Adresse ungültig oder nicht vorhanden ist (Hard Bounce).",
     "Why has my email been disabled?": "Warum wurde meine E-Mail deaktiviert?",
-    "year": "",
+    "year": "jahr",
```
Comment (lsinger) [inline ghost/i18n/locales/de/portal.json:163]: Also a noun: `Jahr`

Comment (cathysarisky) [reply ghost/i18n/locales/de/portal.json:163]: Hey @lsinger: Some context on month and year - these are the /month and /year strings displayed with a paid plan.  Does that change your feelings about capitalization?  (Maybe it doesn't, but in American English, that would look weird.)

image

Comment (lsinger) [reply ghost/i18n/locales/de/portal.json:163]: @cathysarisky thanks for checking back! Yes, that still needs to be capitalized in German -- every noun needs to be. 😅 I've opened a PR with my notes from this one and then a few others here: #21165.

**Pull request merged**